src/analyzer/plugins/analyzerqueenmarybeats.cpp

```diff
@@ -9,9 +9,16 @@
 
 namespace mixxx {
 namespace {
-// stepsize should be equivalent to ~10ms and this constant (11.61ms) make the number 
-// of samples an exact power of 2 for 22.1k, 44.1k, and very close to, for 96k and 192k
-constexpr float kStepSecs = 0.01161; 
+
+// This sets the resolution of the resulting BeatMap.
+// ~12 ms (86 Hz) is a fair compromise between accuracy and analysis speed.
+// These are the preferred window/step sizes from the BeatTrack VAMP
+// With a 44.1 kHz Track we go in 512 Sample Steps
+// TODO: The waveform sample rate of 441 (defined in AnalyzerWaveform::initialize)
```

Comment:
This comment is about AnalyzerWaveform and should be moved there. Why is  it mentioned here?

Reply:
We have two values, kStepSecs here and 441 in AnalyzerWaveform. Because they don't align well, we get interferences. I will update the comment.

Reply:
It is about kStepSecs AND waveform sample rate. I have improved the comment.